zipline/testing/fixtures.py

```diff
@@ -1001,15 +1011,31 @@ def make_bcolz_daily_bar_rootdir_path(cls):
     def init_class_fixtures(cls):
         super(WithBcolzEquityDailyBarReader, cls).init_class_fixtures()
 
+        if cls.BCOLZ_DAILY_BAR_COUNTRY_CODE is None:
```

Comment:
Same Q here re: using a classproperty for this.

Reply:
I'd probably call this `BCOLZ_EQUITY_DAILY_BAR_COUNTRY_CODE` to distinguish from the futures case.

Reply:
The `BCOLZ_DAILY_BAR_` prefix keeps things consistent with the existing attributes on this class, do you think it's worth changing all of them?